python/tchannel/tornado/request.py

```diff
@@ -37,6 +42,9 @@ class Request(object):
     Request class is used to represent the CallRequestMessage at User's level.
     This is going to hide the protocol level message information.
     """
+    MAX_RETRY_TIMES = 3
+    TIMEOUT = 1  # 1 second
+    RETRY_DELAY = 0.3  # 300 ms
```

Comment:
They are configurable in the send method. Use can specify those attributes in the send method.